include/osquery/query.h

```diff
@@ -455,10 +469,13 @@ class Query {
    *
    * @param qd the QueryData object, which has the results of the query.
    * @param epoch the epoch associatted with QueryData
+   * @param counter query execution counter for the specified epoch.
```

Comment:
nit: 'counter the query ...', looks like that would match up with the above params?